tests/smoke_test.go

```diff
@@ -55,10 +55,13 @@ func TestShuffleAppDriverOutput(t *testing.T) {
 		return nil
 	})
 	if err != nil {
-		t.Error(err)
+		t.Fatal(err)
```

Comment:
You may want to [read this thread](https://github.com/mesosphere/kudo-cassandra-operator/pull/3#discussion_r339744459) as there are alternate solutions for handling Errors